Osnova.Net/Constants/DtfSubsiteIds.cs

```diff
@@ -0,0 +1,482 @@
+﻿namespace Osnova.Net.Constants
+{
+    static class DtfSubsiteIds
```

Comment:
Would be great to add link where to find subsite list for the future. Add these as xml-docs for new classes please!